src/ai/backend/common/web/session/__init__.py

```diff
@@ -146,6 +158,20 @@ def __delitem__(self, key: str) -> None:
 STORAGE_KEY = "aiohttp_session_storage"
 
 
+async def update_expires(request: web.Request) -> str:
+    storage = request.get(STORAGE_KEY)
+    if storage is None:
+        raise RuntimeError("Install aiohttp_session middleware in your aiohttp.web.Application")
+    session: Session = await get_session(request)
+    await storage.update_expires(session)
```

Comment:
A `RedisStorage` object updates expiration time of a session because the expiration time is calculated by Redis.
How about let the storage calculate time to extend and implements `update_expires()` (or `update_expiration_time()`) in Session class?